components/hab/src/command/studio/enter.rs

```diff
@@ -300,7 +300,7 @@ mod inner {
 
         for arg in args.iter() {
             let str_arg = arg.to_string_lossy();
-            if str_arg == "-D" {
+            if str_arg == "-D" || str_arg == "-d" {
```

Comment:
If we leave this as it was, the `-d` should pass into the native studio which will then be treated as an error with the changes below.

Reply:
I initially thought this as well. Unfortunately, because powershell variables are case insensitive `-d` does not show up in `$args` instead it sets the `$D` variable. Or am I missing something?

Reply:
I think we can actually ommit `$D` since once you get to the powershell we don't do anything with it. We just need to have the rust strip it off.

Reply:
This is much better thanks. Addressed in [27e558](https://github.com/habitat-sh/habitat/pull/7585/commits/27e558c4c733781eefa76f70a428cf0d92d493fe)